CHANGELOG.md

```diff
@@ -7,6 +7,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+
+- LocationCountry
+  - Guesses country from store settings
```

Comment:
```suggestion
- Guesses country from store settings
```
Is this intentional?

Reply:
Yes, this was intentional because it is `LocationCountry` who does that, but I thought I was going to add other points, I could rewrite it better 👍